test/doc/test_common.py

```diff
@@ -85,7 +84,10 @@ def test_gather_tests():
             ),
         )
         assert isinstance(result, list)
-        assert len(list_expected_types) == len(result)
+
+        # @mmatera this test is failing, why?
```

Comment:
The test fails because in line 49 the message is missing. Then, the text in that and the subsequent lines generates a `DocText` entry that should not be there.